src/app.types.tsx

```diff
@@ -121,19 +122,19 @@ export interface ErrorParsing {
 }
 
 interface Address {
-  building_number: string;
-  street_name: string;
-  town?: string;
-  county?: string;
+  address_line: string;
+  town: string | null;
+  county: string | null;
   postcode: string;
+  country: string;
 }
 
 interface EditAddress {
-  building_number?: string;
-  street_name?: string;
-  town?: string;
-  county?: string;
-  postcode?: string;
+  address_line: string | null;
+  town: string | null;
+  county: string | null;
+  postcode: string | null;
+  country: string | null;
```

Comment:
These should also have an `?` and only have `| null` if they are actually optional.